examples/connext_dds/asynchronous_publication/py/async_publisher.py

```diff
@@ -0,0 +1,126 @@
+#
```

Comment:
I think you should base the example on the C# one, which is more complete:

- it provides an option to use XML or programmatic Qos
- It uses WaitForAsyncPublication